tests/jetty-test-session-common/pom.xml

```diff
@@ -0,0 +1,143 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.eclipse.jetty.tests</groupId>
+    <artifactId>tests</artifactId>
+    <version>12.0.0-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>jetty-test-session-common</artifactId>
+  <name>Tests :: Session Test Tools</name>
+  <packaging>jar</packaging>
+
+  <properties>
+    <bundle-symbolic-name>${project.groupId}.session.test.tools</bundle-symbolic-name>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>jakarta.servlet</groupId>
```

Comment:
Can't be common if it depends on servlet 6 api?

Reply:
I was not sure about this. but it's marked as provided and optional.
It's only few classes from servlet spec.
the real artifact version is used in each ee9/ee10 module

Reply:
changed